<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialogLocate</class>
 <widget class="QDialog" name="TDialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>定位单元格并设置文字</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>列号</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Col">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkColumnInc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>80</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>列增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetFont">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置文字</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/506.bmp</normaloff>:/Pics/images/506.bmp</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>行号</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Row">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkRowInc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>80</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>行增</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>文字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/132.bmp</normaloff>:/Pics/images/132.bmp</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>TDialogLocate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
